logviewer/src/main/java/com/nodesagency/logviewer/domain/FilterState.kt

```diff
@@ -0,0 +1,18 @@
+package com.nodesagency.logviewer.domain
+
+sealed class FilterState(
```

Comment:
It is especially important to document how each filter is expected to behave, i.e. if the query needs to match the string exactly or if it's enough that they're similar in a way. Since we're using the SQL `LIKE` internally, every other repository implementation should also be expected to behave that way. E.g. you only need to enter a substring of a tag and it will find all tags containing that substring.

Reply:
Docs are added and class is renamed 👍